.ado/pipelines/azure-deploy-azureloadtest.yaml

```diff
@@ -0,0 +1,44 @@
+name: '[${{ parameters.environment }}] $(BuildDefinitionName).$(DayOfYear)$(Rev:.r)'
+
+trigger: none
+
+pr: none
+
+pool:
+  vmImage: 'ubuntu-20.04'
+
+parameters:
+- name: environment
+  displayName: 'Environment (Subscriptions)'
+  type: string
+  default: 'int'
+  values:
+  - int
+- name: runLoadTesting # for testing only will later disappear here
+  displayName: "Run embedded load testing"
+  type: boolean
+  default: false
+- name: destroyInfra # option to unselect destroy infrastructure
```

Comment:
What does it mean "option to unselect destroy infrastructure"?

Reply:
This pipeline is not final, yet. I'm mainly using it to test the load test process outside of e2e. It will be aligned with what we have at some point to have a pipeline to spin up a standalone env with service + load test (definition) as well as an embedded version.